### PR TITLE
New function for reading data pulled by the new webscraping

### PR DESCRIPTION
**Pull request by nmmarquez**
Access the back up location of webscraped results to pull data into memory. This data should match the Google Sheet as well as give users access to historical data.

### CODE REVIEW

**Comment by erika-tyagi** [comment]
Looks good to me, but I'll let @hjohns12 approve and merge since she knows more about how the package works than me! Thanks again for doing this!

**Comment by hjohns12** [comment]
Couple questions/thoughts before merging this branch: 

Can `load_latest_data` use the functions `read_fac_info` and/or `clean_facility_name` within the behindbarstools? Want to eliminate the risk that our merges are producing unexpected results, and I think all of us fine-tuning the same functions within behindbarstools is the best way to do this. If that means re-tooling `read_fac_info` and/or `clean_facility_name`, I would rather do that then re-write a close-but-not-quite the same version in this function.

Do you trust `ID` enough to be merging on it at this point? Michael's comment about the number of unique IDs versus facility names made me think not, but curious to hear your thoughts @nmmarquez

**Comment by nmmarquez** [comment]
Regarding the first point on using read_fac_info and clean_facility_name I am in complete agreement, ill work on that today.

For the second point to my knowledge merging on ID is fine the issue is that some facilities correspond to several IDs. That is going to be a merge issue whether we use ID because the Name value will also be the same.

File: R/clean_facility_name.R [hunk 1]
```diff
@@ -5,75 +5,90 @@
...
-              by = c("scrape_name_clean" = "xwalk_name_raw")) %>%
-    hoist(name_xwalk, Name = pluck("xwalk_name_raw", 1)) %>%
-    mutate(Name = map(Name, first),
-          Name = as.character(Name),
-          Name = ifelse(is.na(Name), scrape_name_clean, Name))
-  nrow_federal <- nrow(federal)
-  if(nrow_federal == 0) {federal <- NULL}
+    federal <- dat %>%
+        filter(federal_bool) %>%
+        select(-State) %>%
```
Comment (hjohns12) [inline R/clean_facility_name.R:74]: Implicit here is that we trust the "State" from the fac_spellings sheet more than "State" in the scraped data, right?

**Comment by hjohns12** [comment]
This looks awesome! Obviously we need to discuss the case where State != “Federal” as a data team, but I think it's ready to merge

**Pull request merged**